src/api-service/__app__/onefuzzlib/autoscale.py

```diff
@@ -1,151 +1,143 @@
-#!/usr/bin/env python
-#
-# Copyright (c) Microsoft Corporation.

```

Comment:
was it intentional to remove this ?

Reply:
Yes.  It's now in onefuzzlib/autoscale.py